### PR TITLE
Added Bubble Sort in TypeScript

### LINKED ISSUE #486: Bubble Sort in Typescript
This is an open issue for the Bubble Sort algorithm implementation in Typescript. If you wish to contribute, just make some noise and we will assign you.

### PR DESCRIPTION
**Pull request by Wei-J-Huang**
**Fixes #486 **

By submitting this pull request I confirm I've read and complied with the below declarations.

-   [x] I have read the [Contribution guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md) 
and I am confident that my PR reflects them.

-   [x] I have followed the [coding guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#cs) 
for this project.

-   [x] My code follows the [skeleton code structure](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#sample).

-   [x] This pull request has a descriptive title. 
For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.

-   [x] This pull request shall only be merged once the build passes.

-   [x] I have mentioned the issue number correctly (with hyper-link) in this pull request description.

- I noticed that the JavaScript version already exists, to keep consistency, the codes were reused. Please let me know if there are any changes to be made

### CODE REVIEW

File: algorithms/ar-bsrt/typescript/bubble_sort.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            }
+        }
+    } while (swapped);
+    return arr;
+}
+
+//Function to test bubbleSort
+let main = () => {
+	let arr = [7, 9, 5, 8, 2, 3, 1, 6, 4];
+    console.log(bubbleSort(arr));
```
Comment (RavinduSachintha) [inline algorithms/ar-bsrt/typescript/bubble_sort.ts:22]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6329238)

Comment (Wei-J-Huang) [reply algorithms/ar-bsrt/typescript/bubble_sort.ts]: console.log line has been replaced by simply calling the function and passing the array, please check again :)

**Review by RavinduSachintha — APPROVED** [review]

**Comment by RavinduSachintha** [comment]
@Wei-J-Huang would you be able to add this contribution to the algorithm chart on the main readme file? Don't change anything else. And also don't make another PR. just push changes to here. :+1:

**Comment by Wei-J-Huang** [comment]
README change should be done, let me know if there’s any issues.

**Review by RavinduSachintha — APPROVED** [review]

**Comment by RavinduSachintha** [comment]
@all-contributors please add @Wei-J-Huang for code and doc

**Comment by RavinduSachintha** [comment]
@Wei-J-Huang good job :+1: keep it up.
Make sure to star this repo :wink:

**Pull request merged**